magpylib/_src/input_checks.py

```diff
@@ -9,6 +9,19 @@
 )
 
 
+def check_field_input(inp, origin):
```

Comment:
field=True, allowed? Why?

Reply:
Doesn't feel right to hand the strings `B` and `H`around library internally. It feels more like a user feature.
This way we get to keep the bool `bh` from level 3 to level 1.